### PR TITLE
EPMRPP-81506 || Implement Datepicker

### PR DESCRIPTION
**Pull request by Bam6ycha**

### CODE REVIEW

**Review by Vadim73i — APPROVED** [review]

File: app/src/componentLibrary/datePicker/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- **customClassName** : _string_, optional, default : ""
+- **customTimeInput** : _node_, optional, default = null
+- **shouldCloseOnSelect** : _bool_, optional, default = false
+- **showPopperArrow** : _bool_, optional, default=false
+- **popperClassName** : _string_, optional, default = ""
+- **calendarClassName** : _string_, optional, default = ""
+- **fixedHeight** : _bool_, optional, default = false
+- **children** : _node_, optional, default = null
+- **headerNodes** : _node_, optional, default = null
+- **value** : _string_, optional, default = new Date()
```
Comment (AmsterGet) [inline app/src/componentLibrary/datePicker/README.md:20]: I don't see the default `value` in the `DatePicker.defaultProps`.

File: app/src/componentLibrary/datePicker/header/datePickerHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  prevMonthButtonDisabled,
+  nextMonthButtonDisabled,
+  headerNodes,
+  customClassName,
+}) => {
+  const { formatMessage } = useIntl();
+  const currentYear = date.getFullYear();
+  const currentMonth = date.getMonth();
+
+  const mothsDropdownOptions = useMemo(
```
Comment (AmsterGet) [inline app/src/componentLibrary/datePicker/header/datePickerHeader.jsx:53]: month

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/componentLibrary/datePicker/header/datePickerHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+import { Dropdown } from 'componentLibrary/dropdown';
+import { COMMON_LOCALE_KEYS, months } from 'common/constants/localization';
+import styles from './datePickerHeader.scss';
+import DatePickerHeaderIcon from './img/calendar-arrow-inline.svg';
+
+const cx = classNames.bind(styles);
+
+const messagesMonths = Object.fromEntries(
+  Object.entries(COMMON_LOCALE_KEYS).filter(([key]) => months.includes(key)),
+);
```
Comment (AmsterGet) [inline app/src/componentLibrary/datePicker/header/datePickerHeader.jsx:31]: Can you extract the months simpler?
F.e. like here

image

File: app/src/componentLibrary/datePicker/header/datePickerHeader.jsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  prevMonthButtonDisabled,
+  nextMonthButtonDisabled,
+  headerNodes,
+  customClassName,
+}) => {
+  const { formatMessage } = useIntl();
+  const currentYear = date.getFullYear();
+  const currentMonth = date.getMonth();
+
+  const mothDropdownOptions = useMemo(
```
Comment (AmsterGet) [inline app/src/componentLibrary/datePicker/header/datePickerHeader.jsx:53]: month

File: app/src/componentLibrary/datePicker/header/datePickerHeader.jsx [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+  );
+
+  const yearDropdownOptions = useMemo(() => {
+    const yearValues = getYearsFrom(currentYear);
+    return yearValues.reduce((acc, year) => acc.concat({ value: year, label: year }), []);
+  }, [currentYear]);
+
+  const currentYearDisplayed = yearDropdownOptions.find(({ value }) => value === currentYear);
+
+  const currentMothDisplayed = mothDropdownOptions[currentMonth];
```
Comment (AmsterGet) [inline app/src/componentLibrary/datePicker/header/datePickerHeader.jsx:71]: month

**Review by AmsterGet — CHANGES_REQUESTED** [review]
Also please update message ids for Chinese localization

**Pull request merged**